bin/Powheg/production/Run3/PbPb_5p36TeV/WToLNu/WmToENu_EPPS21.input

```diff
@@ -0,0 +1,62 @@
+! W boson production parameters
+idvecbos   -24     ! PDG code for vector boson to be produced ( W+:24 W-:-24 )
+vdecaymode   1     ! code for selected W decay (1: electronic; 2: muonic; 3: tauonic)
+withdamp     1     ! (default 1, use) use Born-zero damping factor
+
+numevts NEVENTS    ! number of events to be generated
+ih1   1            ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1            ! hadron 2 (1 for protons, -1 for antiprotons)
+ebeam1 2681d0      ! energy of beam 1
+ebeam2 2681d0      ! energy of beam 2
+! To be set only if using LHA pdfs
+lhans1   904400    ! nPDF: EPPS21nlo_CT18Anlo_Pb208
+lhans2   904400    ! nPDF: EPPS21nlo_CT18Anlo_Pb208
+
+! Parameters to allow or not the use of stored data
+use-old-grid    1  ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1  ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+ncall1   500000    ! number of calls for initializing the integration grid
+itmx1    5         ! number of iterations for initializing the integration grid
+ncall2   500000    ! number of calls for computing the integral and finding upper bound
+itmx2    5         ! number of iterations for computing the integral and finding upper bound
+foldcsi  1         ! number of folds on csi integration
+foldy    1         ! number of folds on  y  integration
+foldphi  1         ! number of folds on phi integration
+nubound  50000     ! number of bbarra calls to setup norm of upper bounding function
+icsimax  1         ! <= 100, number of csi subdivision when computing the upper bounds
+iymax    1         ! <= 100, number of y subdivision when computing the upper bounds
+xupbound 2d0       ! increase upper bound for radiation generation
+
+! OPTIONAL PARAMETERS
+
+renscfact  1d0     ! (default 1d0) ren scale factor: muren  = muref * renscfact
+facscfact  1d0     ! (default 1d0) fac scale factor: mufact = muref * facscfact
+testplots  1       ! (default 0, do not) do NLO and PWHG distributions
```

Comment:
Why ?

Reply:
(just curiosity.)

Reply:
I think I took it from the other Powheg cards. Do you recommend removing it?

Reply:
I think only `testplots` is fishy, the other `muren` `mufact` scale factors are just normal

But for `testplots` IIRC it should be the topdraw hists that is computationally cheap and compact in size so perhaps fine

Reply:
ok, I will remove the testplots in the powheg cards